src/utilities/apiCalls.js

```diff
@@ -0,0 +1,61 @@
+export const fetchMovies = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => {
+      if (!response.ok) {
+        throw Error()
+      }
+      return response.json()
+    })
+}
+
+export const fetchMovie = (id) => {
```

Comment:
You are exporting the same variable twice! I am also thinking this could be a bit more dynamic anyways

EDIT: Or we might need just different variable names for easier distinction between he two functions